### ISSUE TITLE
[Error] Support for RTX 30-series

### ISSUE THREAD

**Issue by pfvaldez** [NVlabs/UMR#9]
Hi Xueting

Thank you for your work. I tried running the demo but I cannot run it on my machine with RTX 3070, I was wondering if there is a fix for this?

I am using CUDA 11

```

> $ python -m UMR.experiments.demo --model_path UMR/cachedir/snapshots/cub_net/pred_net_latest.pth --img_path UMR/demo_imgs/birdie.jpg --out_path UMR/cachedir/demo
> Traceback (most recent call last):
>   File "/home/miniconda3/envs/umr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
>     "__main__", mod_spec)
>   File "/home/miniconda3/envs/umr/lib/python3.6/runpy.py", line 85, in _run_code
>     exec(code, run_globals)
>   File "/home/Development/UMR/experiments/demo.py", line 32, in <module>
>     from ..nnutils import test_utils
>   File "/home/Development/UMR/nnutils/test_utils.py", line 42, in <module>
>     import soft_renderer as sr
>   File "/home/miniconda3/envs/umr/lib/python3.6/site-packages/soft_renderer-1.0.0-py3.6-linux-x86_64.egg/soft_renderer/__init__.py", line 1, in <module>
>     from . import functional
>   File "/home/miniconda3/envs/umr/lib/python3.6/site-packages/soft_renderer-1.0.0-py3.6-linux-x86_64.egg/soft_renderer/functional/__init__.py", line 4, in <module>
>     from .load_obj import load_obj
>   File "/home/miniconda3/envs/umr/lib/python3.6/site-packages/soft_renderer-1.0.0-py3.6-linux-x86_64.egg/soft_renderer/functional/load_obj.py", line 7, in <module>
>     import soft_renderer.cuda.load_textures as load_textures_cuda
> ImportError: /home/miniconda3/envs/umr/lib/python3.6/site-packages/soft_renderer-1.0.0-py3.6-linux-x86_64.egg/soft_renderer/cuda/load_textures.cpython-36m-x86_64-linux-gnu.so: undefined symbol: _ZN6caffe26detail36_typeMetaDataInstance_preallocated_7E

```

**Comment by pfvaldez** [comment]
As an update, I was able to somehow fix it by removing `soft_renderer-1.0.0-py3.6-linux-x86_64.egg/soft_renderer` and re-installing it. However another error appears

```

> $ python -m UMR.experiments.demo --model_path UMR/cachedir/snapshots/cub_net/pred_net_latest.pth --img_path UMR/demo_imgs/birdie.jpg --out_path UMR/cachedir/demo
> Loaded model from UMR/cachedir/snapshots/cub_net/pred_net_latest.pth.
> Read image from UMR/demo_imgs/birdie.jpg.
> THCudaCheck FAIL file=/pytorch/aten/src/THC/THCGeneral.cpp line=383 error=8 : invalid device function
> Traceback (most recent call last):
>   File "/home/pfvaldez/anaconda3/envs/umr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
>     "__main__", mod_spec)
>   File "/home/pfvaldez/anaconda3/envs/umr/lib/python3.6/runpy.py", line 85, in _run_code
>     exec(code, run_globals)
>   File "/home/pfvaldez/Development/UMR/experiments/demo.py", line 221, in <module>
>     app.run(main)
>   File "/home/pfvaldez/anaconda3/envs/umr/lib/python3.6/site-packages/absl/app.py", line 300, in run
>     _run_main(main, args)
>   File "/home/pfvaldez/anaconda3/envs/umr/lib/python3.6/site-packages/absl/app.py", line 251, in _run_main
>     sys.exit(main(argv))
>   File "/home/pfvaldez/Development/UMR/experiments/demo.py", line 218, in main
>     tester.demo()
>   File "/home/pfvaldez/Development/UMR/experiments/demo.py", line 112, in demo
>     outputs = self.model.forward(self.input_imgs)
>   File "/home/pfvaldez/Development/UMR/nnutils/cub_mesh.py", line 453, in forward
>     img_feat, noise, mean, logvar = self.encoder(img)
>   File "/home/pfvaldez/anaconda3/envs/umr/lib/python3.6/site-packages/torch/nn/modules/module.py", line 493, in __call__
>     result = self.forward(*input, **kwargs)
>   File "/home/pfvaldez/Development/UMR/nnutils/cub_mesh.py", line 109, in forward
>     resnet_feat = self.resnet_conv(img)
>   File "/home/pfvaldez/anaconda3/envs/umr/lib/python3.6/site-packages/torch/nn/modules/module.py", line 493, in __call__
>     result = self.forward(*input, **kwargs)
>   File "/home/pfvaldez/Development/UMR/nnutils/cub_mesh.py", line 63, in forward
>     x = self.resnet.relu(x)
>   File "/home/pfvaldez/anaconda3/envs/umr/lib/python3.6/site-packages/torch/nn/modules/module.py", line 493, in __call__
>     result = self.forward(*input, **kwargs)
>   File "/home/pfvaldez/anaconda3/envs/umr/lib/python3.6/site-packages/torch/nn/modules/activation.py", line 99, in forward
>     return F.relu(input, inplace=self.inplace)
>   File "/home/pfvaldez/anaconda3/envs/umr/lib/python3.6/site-packages/torch/nn/functional.py", line 941, in relu
>     result = torch.relu_(input)
> RuntimeError: CUDA error: no kernel image is available for execution on the device

```

**Comment by sunshineatnoon** [comment]
Hi, I believe this is an issue with the soft rasterizer, please make sure that it can run on the RTX 30-series first.
